### PR TITLE
Support SwiftPM

### LINKED ISSUE #14: Support Swift Package Manager
PMKUIKit doesn't support Swift Package Manager. This requires a Package.swift file at the root of the repository following the format defined in [Package Description](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md).

### PR DESCRIPTION
**Pull request by lstomberg**
Fixes #14

### CODE REVIEW

**Comment by lstomber** [comment]
Any chance at getting this merged and a new tag created so that this project supports SPM?

File: Sources/UIImagePickerController+Promise.swift [hunk 1]
```diff
@@ -5,10 +5,21 @@ import UIKit
 
 #if !os(tvOS)
 
+public struct AnimationOptions: OptionSet {
```
Comment (RomanPodymov) [inline Sources/UIImagePickerController+Promise.swift:8]: Hello @lstomberg 
Why do you need to re-define `PMKAnimationOptions`?

**Review by RomanPodymov — CHANGES_REQUESTED** [review]

**Comment by RomanPodymov** [comment]
Hello @mxcl 
Can you check this, please?

File: Sources/UIImagePickerController+Promise.swift (hunk 1 above)
Comment (mxcl) [reply Sources/UIImagePickerController+Promise.swift:8]: Presumably for SwiftPM since it won't get the objective-c portions

**Comment by mxcl** [comment]
Looks fine

**Review by RomanPodymov — APPROVED** [review]

**Comment by Vyazovoy** [comment]
Hi, @RomanPodymov! Could you please make it work with the current PromiseKit version (currently it 6.8.3<7.0.0) and add a new version tag?

**Comment by RomanPodymov** [comment]
Hello @Vyazovoy 
Added v.4.0.0.

**Comment by Vyazovoy** [comment]
Thanks for so quick update, @RomanPodymov! 🙏

**Pull request merged**